### PR TITLE
GitHub Flow with Circle and Heroku

### PR DESCRIPTION
**Pull request by ztolley**
A document to outline how to use GitHub Flow, Heroku and Circle CI together to make continuous easy.

### CODE REVIEW

**Comment by rgarner** [comment]
The commit still says "WIP" – is it?

File: devops/githubflow.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+- Source code management
+- Testing
+- Deployment
+
+### Source Code Management
+At the heart of this approach is the use of an approach to source code change management named [GitHub Flow](https://guides.github.com/introduction/flow/).
+
+![GitHub Flow](https://d2mxuefqeaa7sj.cloudfront.net/s_28CBF72EEB7B1FA91E0376EDB3A58CF7FBD0FAC3086623F971AC963E9A3C4C86_1505228283428_download.png)
+
+In GitHub Flow you have one central branch of code, the ‘Master’ branch. Master represents the code that is currently live, the code you can trust. Do NOT check broken or incomplete code into Master EVER! Bad things will happen, if you are lucky you’ll only loose your job.
```
Comment (sl33p) [inline devops/githubflow.md]: I would leave the losing your job jokes out, but if it really must be kept it should be spelled correctly.

**Comment by sl33p** [comment]
I asked Joel to take a quick look too. Looks really comprehensive. :)

**Comment by rgarner** [comment]
Content cherry-picked to [ops-manual](https://github.com/barnardos/ops-manual)

**Pull request closed without merging**